examples/prometheus/prometheus.yaml

```diff
@@ -643,6 +647,66 @@ objects:
           action: replace
           target_label: kubernetes_name
 
+      # Scrape aggregated logging endpoints.
+      #
+      # The relabeling allows the actual service scrape endpoint to be configured
+      # via the following annotations:
+      #
+      # * `prometheus.io/scrape`: Only scrape services that have a value of `true`
+      # * `prometheus.io/scheme`: If the metrics endpoint is secured then you will need
+      # to set this to `https` & most likely set the `tls_config` of the scrape config.
+      # * `prometheus.io/path`: If the metrics path is not `/metrics` override this.
+      # * `prometheus.io/port`: If the metrics are exposed on a different port to the
+      # service then set this appropriately.
+      - job_name: 'kubernetes-logging-service-endpoints'
+
+        tls_config:
+          ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+          # TODO: this should be per target
```

Comment:
Can we get this done as part of this commit?

Reply:
Oh... I cut&pasted this comment from the same configuration in existing `kubernetes-service-endpoints` job. Will remove this comment, it is confusing.